nashville-software-school/Class-Website
# Cj site address fix

## Link to Ticket

TITLE [Link]()

## Description of Proposed Changes
My portfolio link is going to "https://nss-day-cohort-46.github.io/calebsjames.com" instead of "www.calebsjames.com"
I added in "http://www." at the beginning of the address.

Also, my fun photo was not showing up.
- 

## Steps to Test

Click on my portfolio link, it should go to calebsjames.com
Click on my professional photo to see the fun photo with the bio

```sh
git fetch --all
git checkout <cj-SiteAddressFix>

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Only my card

## Mentions @username

Tag users that need to review this code
